components/src/core/Drawer/DrawerHeader.tsx

```diff
@@ -100,7 +100,6 @@ const useStyles = makeStyles((theme: Theme) =>
             marginLeft: theme.spacing(1),
         },
         subtitle: {
-            fontWeight: 300,
```

Comment:
Font-weight 300 is light, I believe. Are we using regular (400) in the DSM?

Reply:
Yeah, it's 400 in the DSM. I couldn't find a formal design for the DrawerHeader, but the appbar pattern has 400 font-weight for the subtitle and Huyaun confirmed that it is using Body2 which is 14px 400 font-weight.